packages/flutter/lib/src/material/icons.dart

```diff
@@ -21,6 +21,27 @@ import 'package:flutter/widgets.dart';
 ///   uses-material-design: true
 /// ```
 ///
+/// {@tool snippet --template=stateless_widget_scaffold}
+///
+/// This example shows how to use [Icons] to create three icons: fast rewind,
```

Comment:
`This example creates three [Icons], fast_rewind, play_arrow and fast_forward, in that order.`